### DISCUSSION TITLE
qrqp demo

### DISCUSSION THREAD

**Discussion by srivats2794** [casadi/casadi#3094] [General]
Hi. I'm trying to run the qrqp demo in this link https://github.com/jgillis/mpc-simulink-demo

When I run the matlab script, it throws this error for the mex command:

```
MPC_step.obj : error LNK2019: unresolved external symbol MPC_step_qp_print_header referenced in function MPC_step_f3
MPC_step.obj : error LNK2019: unresolved external symbol MPC_step_qp_print_iteration referenced in function MPC_step_f3
LMPC.mexw64 : fatal error LNK1120: 2 unresolved externals
```

I'm able to see those functions in the auto-generated MPC_step.c script. Can anyone help me resolve this please?

**Comment by srivats2794** [comment]
@jgillis I see its you who has provided that example. Thank you. It is very helpful for LMPC. I thought it'd be appropriate to tag you since its your code.

**Comment by jgillis** [comment]
It seems 3.6.0 introduced a printing issue.
To work around it, remove "-DCASADI_PRINTF=ssPrintf" from the mex command in `generate_block.m`
